sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala

```diff
@@ -543,7 +543,7 @@ object IntegratedUDFTestUtils extends SQLHelper {
         |    def analyze(initial_count, input_table):
         |        buffer = ""
         |        if initial_count.value is not None:
-        |            assert(not initial_count.is_table)
+        |            assert(not initial_count.isTable)
         |            assert(initial_count.data_type == IntegerType())
```

Comment:
`data_type` should be `dataType`

Reply:
`with_single_partition`, `partition_by` and `order_by` should also be changed accordingly.